### ISSUE TITLE
[CLOSED] move decay between transactions into the transactions

### ISSUE THREAD

**Issue by ulfgebhardt** [Zahlungsmittel/Zahlungsmittel#542]
<a href="https://github.com/einhornimmond"><img src="https://avatars.githubusercontent.com/u/67117805?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [einhornimmond](https://github.com/einhornimmond)**
_Monday Jun 07, 2021 at 08:42 GMT_
_Originally opened as https://github.com/gradido/gradido/pull/483_

----

## 🍰 Pullrequest
- Move decay between transactions into the transactions
- Remove second decay from creation transaction

### Issues
- fixes #479 
- fixes #318
- fixes #369 
- fixes #582

### Todo
- [x] Wait on merging Login-Coverage (#472) to update Community-Server tests for this new feature

----

_**[einhornimmond](https://github.com/einhornimmond)** included the following code: https://github.com/gradido/gradido/pull/483/commits_

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/ulfgebhardt"><img src="https://avatars.githubusercontent.com/u/1238238?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ulfgebhardt](https://github.com/ulfgebhardt)**
_Tuesday Jun 08, 2021 at 21:44 GMT_

----

I would like to postpone the merge of this till we find a way to display the decay properly in the frontend once its not its own transaction anymore. If we blindly merge this, we will/might have a communication problem.
